internal/pkg/ccloudv2/utils.go

```diff
@@ -24,7 +24,7 @@ type NullableString interface {
 	IsSet() bool
 }
 
-var Hostnames = []string{"confluent.cloud", "cpdev.cloud"}
+var Hostnames = []string{"confluent.cloud", "cpdev.cloud", "confluentgov.com", "confluentgov-internal.com"}
```

Comment:
as cl is source available now, is `-internal` something ok to be exposed to customers?

Reply:
I think it's fine. We include references to `cpdev.cloud` throughout the code, and I'd say this is similar.

Reply:
@brianstrauch Could you help review this code again?

Reply:
The code looks fine, but I'd like to test it before merging. Are you able to create a login for me on confluentgov.com so I can give this a try?

Reply:
> we need to make it ready for use before merging the code?

Yep, that would be great! If there's an issue with it, I want to know about it before we release buggy code :smile:

Reply:
however, It requires a lot of other changes to support okta-based sso in CLI. Do I need to make it to be a huge PR?

Reply:
> It requires a lot of other changes to support okta-based sso in CLI.

We already support Okta-based SSO in the CLI for our other three environments, so I'm hopeful that this should be relatively easy to add! Even if the PR is a bit bigger, it would be great to make PRs that add 100% of the functionality (instead of 50%).